src/pegout/components/PegoutForm.vue

```diff
@@ -429,8 +429,7 @@ export default defineComponent({
         .then(() => getFlyoverProviders())
         .then(() => {
           appendRecaptcha(flyoverService.value.siteKey);
-        })
-        .catch(console.error);
+        });
```

Comment:
This will be appropriately handled [here](https://github.com/rsksmart/2wp-app/pull/891).